gluoncv/logs/classification/imagenet/vgg16_bn.sh

```diff
@@ -0,0 +1 @@
+python train_imagenet.py --batch-size 32 --num-gpus 8 -j 64 --mode hybrid --num-epochs 100 --lr 0.01 --lr-decay 0.1 --lr-decay-epoch 50,80 --batch-norm --model vgg16_bn
```

Comment:
`--batch-norm` is only for vgg16_bn, but not vgg19_bn ?

Reply:
`--batch-norm` is not required because its set to True when a _bn model is loaded acc [here](https://github.com/apache/incubator-mxnet/blob/b16f87505a63aa177854f1aed93bf38e263b6857/python/mxnet/gluon/model_zoo/vision/vgg.py#L246)

For the sake of consistency I can either add `--batch-norm` for both vgg16_bn and 19_bn or remove for both. What do you think?

Reply:
Please remove them.

Reply:
done